pkg/configuration/base/resources/resources_test.go

```diff
@@ -0,0 +1,229 @@
+package resources
+
+import (
+	"testing"
+
+	"github.com/jenkinsci/kubernetes-operator/pkg/apis/jenkins/v1alpha2"
+	"github.com/stretchr/testify/assert"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+var jenkins = v1alpha2.Jenkins{
+	Spec: v1alpha2.JenkinsSpec{
+		Master: v1alpha2.JenkinsMaster{
+			Containers: []v1alpha2.Container{
+				{
+					Env: []corev1.EnvVar{},
+					ReadinessProbe: &corev1.Probe{
+						Handler: corev1.Handler{
+							HTTPGet: &corev1.HTTPGetAction{},
+						},
+					},
+					LivenessProbe: &corev1.Probe{
+						Handler: corev1.Handler{
+							HTTPGet: &corev1.HTTPGetAction{},
+						},
+					},
+				},
+			},
+		},
+	},
+}
```

Comment:
reduce indents

Reply:
They're normal. It's an illusion because of github formatting.